### ISSUE TITLE
Append "(Day x/y)" to the end of event title of multi day events

### ISSUE THREAD

**Issue by shompoe** [andstatus/todoagenda#81]
I borrowed this idea from Google calendar so just create a multi day event and look at it in Google calendar to see what I mean.

**Comment by yvolk** [comment]
Logically "Day 3/5" looks more like a part of the event's date than of its title.
We have e.g. "in 3 days" there. See https://github.com/andstatus/todoagenda/issues/7

@shompoe What do you think?

**Comment by shompoe** [comment]
Indeed.      Personally I prefer to keep the event start/finish time underneath the event title and to see the day of week and date as part of header. So in my case a "Day x/y" inline with the event title would be very clear in it's meaning.  But perhaps I forgot that it might cause confusion in other configurations.

So I leave it to your disgretion. If you think it is a bad idea then don't do it.




9 Nov 2021, 06:29 by ***@***.***:

**Comment by yvolk** [comment]
@shompoe I see that "Day x/y" is really NOT a formatted date, this is a formatted date in the date period. Meaning that we need three dates instead of one in order to create the formatted "day of event period".
So configuring and placing it separately from the formatted date is normal...

Thank you for the suggestion, I will think about this feature.

**Comment by yvolk** [comment]
I see that Google calendar does something like this.
Event doesn't need to be an "All day event". It needs to span several days.
![Image](https://github.com/user-attachments/assets/04c5b589-5451-4a94-ac28-049546065a19)

**Comment by yvolk** [comment]
I added another event "Today's event till Monday" that already started:
![Image](https://github.com/user-attachments/assets/4de2786f-5793-4775-a0ec-dab2d10b41c2)

**Comment by yvolk** [comment]
Implemented in v4.12.2

For default "Two rows layout":
![Image](https://github.com/user-attachments/assets/ba2839b9-c16c-4ce1-8cb6-aa8182134890)

**Comment by yvolk** [comment]
The same for "All in one row" Event entry layout (Event entry - Date hidden):

![Image](https://github.com/user-attachments/assets/6ea519a3-6dc4-4a39-9608-3b0d02ac4d57)

**Comment by yvolk** [comment]
In settings:

![Image](https://github.com/user-attachments/assets/3a944a00-e33f-4935-a1e8-41f3933f78aa)
